src/components/terminal/input/terminal-input.component.js

```diff
@@ -15,6 +15,11 @@ export const TerminalInput = props => {
             return;
         }
 
+        element.style.height = element.scrollHeight + "px"
```

Comment:
consider sizing the div in larger blocks instead pixel by pixel for performance and sanity.